### ISSUE TITLE
Cannot launch jupyterlab_iframe

### ISSUE THREAD

**Issue by ljvmiranda921** [timkpaine/jupyterlab_iframe#102]
Hi there, I tried installing jupyterlab_iframe in a Google Cloud Vertex AI notebook. I followed the installation instructions in the README.
Running

```sh
jupyter labextension list
```

returns:

```
(base) jupyter@explosion-test-prodigy:~/test$ jupyter labextension list
Config option `kernel_spec_manager_class` not recognized by `ListLabExtensionsApp`.
[W 2022-01-03 01:41:03.663 LabApp] Config option `kernel_spec_manager_class` not recognized by `LabApp`.
JupyterLab v3.2.5
/opt/conda/share/jupyter/labextensions
        nbdime-jupyterlab v2.1.1 enabled OK
        jupyterlab-prodigy v4.0.0 enabled OK (python, jupyterlab-prodigy)
        jupyterlab-jupytext v1.3.8 enabled OK (python, jupytext)
        jupyterlab_iframe v0.4.1 enabled OK
        @jupyterlab/git v0.34.0 enabled OK (python, jupyterlab-git)
        @jupyterlab/server-proxy v3.2.0 enabled OK
        @jupyter-widgets/jupyterlab-manager v3.0.1 enabled OK (python, jupyterlab_widgets)

Other labextensions (built into JupyterLab)
   app dir: /opt/conda/share/jupyter/lab
        beatrix_jupyterlab v3.1.6 disabled OK
        jupyterlab-plotly v5.4.0 enabled OK
        plotlywidget v4.14.3 enabled OK
        wit-widget v1.8.1 enabled OK
```

Which shows `jupyterlab_iframe` to be enabled. **However, I cannot find the extension in the Launch Menu**. 
Perhaps I'm just missing a keyboard shortcut or something? Thanks a lot! 😄

**Comment by timkpaine** [comment]
It doesn't look liked you installed it, and this isn't enough information to do anything with. It works fine for me in a clean jupyterlab environment following the instructions from the readme.

**Comment by ljvmiranda921** [comment]
Oh interesting, thanks for the reply, will check again :) I had a misconception that if 

```
 jupyterlab_iframe v0.4.1 enabled OK
```

shows up, then it's working. Perhaps something went wrong in between. Thanks again and will reply here once I've solved it 👍

**Comment by timkpaine** [comment]
is it present in the server extensions as well? console logs?  https://stackoverflow.com/help/minimal-reproducible-example

**Comment by ljvmiranda921** [comment]
As a reference, here are the logs during installation. I didn't see anything particularly suspicious:

```
$ jupyter labextension install jupyterlab_iframe
Config option `kernel_spec_manager_class` not recognized by `InstallLabExtensionApp`.
[W 2022-01-03 02:36:17.604 LabApp] Config option `kernel_spec_manager_class` not recognized by `LabApp`.
Building jupyterlab assets (production, minimized)

$ jupyter serverextension enable --py jupyterlab_iframe
Config option `kernel_spec_manager_class` not recognized by `EnableServerExtensionApp`.
Enabling: jupyterlab_iframe.extension
- Writing config: /home/jupyter/.jupyter
    - Validating...
      jupyterlab_iframe.extension  OK
```

Then what I did is refresh my browser, head over to the `Launcher` (Jupyter v3.2.5), then look for the `jupyterlab_iframe` extension. This is where I'm already stuck: I can't find the extension in the menu:

![image](https://user-images.githubusercontent.com/12949683/147896423-a7824f02-c251-4a4e-9a12-51ab3d4b6f97.png)

I also opened JupyterLab's extension manager, and looked for the jupyterlab_iframe. It's grayed out

![image](https://user-images.githubusercontent.com/12949683/147896445-d8de0ff3-6c74-4c54-89a0-a85623739c99.png)

Then when I click the "About" button in the screenshot above, this warning showed up:

> This is a prebuilt extension. To uninstall it, please read the user guide on: 
> https://jupyterlab.readthedocs.io/en/3.2.x/user/extensions.html

![image](https://user-images.githubusercontent.com/12949683/147896471-d1a7f5b3-0c87-4b41-9774-14c3d96a2138.png)

Aside from that, I don't have any other logs to share. I guess one thing I'm doing differently is that I'm running this in a Vertex AI notebook, but I really don't think it affects anything&mdash; Google Cloud will just run a Jupyter lab instance as if I'm running it on my machine.
